stdlib/email/src/main/ballerina/src/email/commons.bal

```diff
@@ -35,11 +36,21 @@ public type Email record {|
     string subject;
     string|xml|json body;
     string contentType?;
+    Header[] headers?;
```

Comment:
Is it possible to make it as map value?
If we have a record array, we need to iterate all the array elements to read the header value.

Reply:
like,
```ballerina
map<string> headers?
```

Reply:
Changed to a map value.